src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1609,6 +1609,7 @@ public function getPreferredFormat(?string $default = 'html'): ?string
     public function getPreferredLanguage(array $locales = null): ?string
     {
         $preferredLanguages = $this->getLanguages();
+        $locales = array_map([$this, 'formatLocale'], $locales ?? []);
```

Comment:
Since PHP 8.1, the project preference is to use first-class callable syntax instead of the array syntax.

```suggestion
        $locales = array_map($this->formatLocale(...), $locales ?? []);
```